GraphQL: Remove reference info from Attributes section

keharper opened the pull request:
## Purpose of this pull request

This pull request (PR) removes the manually maintained reference information from the topics in the pages/graphql/schema/attributes directory.

## Affected pages

<!-- REQUIRED List the affected pages on developer.adobe.com (URLs). Not necessary for large numbers of files. -->

- ...

## Links to Magento Open Source code

<!--  OPTIONAL - REMOVE THIS SECTION IF NOT USED. If this pull request references a file in a Magento Open Source or Adobe Commerce codebase repository, add it here. -->

- ...

<!--
If you are fixing a GitHub issue, using the GitHub keyword format (https://help.github.com/en/articles/closing-issues-using-keywords#closing-an-issue-in-a-different-repository) closes the issue when this pull request is merged. Example: `Fixes #1234`.
`main` is the default branch. Merged pull requests to `main` go live on the site automatically. Any requested changes to content on the `main` branch must be related to the released codebase. Any content related to future releases goes in the `develop` branch.
See Contribution guidelines (https://github.com/AdobeDocs/commerce-webapi/blob/main/.github/CONTRIBUTING.md) for more information.
-->

jeff-matthews commented on src/pages/graphql/schema/attributes/index.md:
```diff
@@ -3,3 +3,5 @@ title: Attributes
 
 # Attributes
+
+The attributes queries allow you to retrieve values stored as custom and [Entity-Attribute-Value (EAV)()] attributes. Custom attributes are those added on behalf of a merchant. For example, a merchant might need to add attributes to describe products, such as shape or volume. EAV attributes are defined by the system.
```
Should the first occurrence of EAV link somewhere? The formatting looks off; like it's meant to use a link definition, but there is no definition in this file.

jeff-matthews commented on src/pages/graphql/schema/attributes/queries/custom-attribute-metadata.md:
```diff
@@ -10,10 +10,16 @@ The `customAttributeMetadata` query has been deprecated. Use the [`customAttribu
 
+## Reference
+
+The [`customAttributeMetadata`](https://developer.adobe.com/commerce/webapi/graphql-api/index.html#query-customAttributeMetadata) reference provides detailed information about the types and fields defined in this query.
```
This is the only link that seems to resolve to a specific section.

jeff-matthews reviewed (commented):
With the exception of the noted URL, none of the other links seem to resolve to a specific reference section. They just take me to the top of the main ref page.

keharper commented:
`attributesForm`, `atttributesList`, and `customAttributeMetadataV2` are new for 2.4.7, so I've replaced the link so that it points to the Beta version of SpectaQL.

`attributesMetadata` was originally defined in PWA, but unlike the `contactUs` mutation and the `recaptchaV3Config` query, it was not ported to core. I presume it's because the query as been deprecated. I've reverted the contents of this topic. I had already created COMDOX-727 to handle PWA. I thought I was going to be able to close it without any work, because the topics would all be updated in the other sections.

I'll do a little investigating and probably add a sentence stating the query is available only if PWA is installed. I'll do that under COMDOX-727.
